### PR TITLE
localize subplot names

### LINKED ISSUE #665: Localized subplot/axis names in subplot panel folds and elsewhere

### PR DESCRIPTION
**Pull request by dmt0**
closes #665 
addresses #631

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Looking good, nice refactoring/localization!

Small things:

1. Pie seems to get its number from the trace number in general, so if the first pie is the 7th trace, the subplot is Pie 6
2. The small icons for cones/streamtubes are still defaulting to the 'line' icon... can we patch things so they get the corresponding temp icons for scatter3d and scatterline please?

image

File: src/components/containers/SubplotAccordion.js [hunk 1]
```diff
@@ -111,23 +118,34 @@ class SubplotAccordion extends Component {
...
-            name={layoutKey}
+            name={subplotName}
           >
             {children}
           </NonCartesianSubplotFold>
         );
       }
     });
 
+    let pieCounter = 0;
```
Comment (nicolaskruchten) [inline src/components/containers/SubplotAccordion.js]: hehe OK, we'll refactor that next time, when we add a third :)

**Comment by nicolaskruchten** [comment]
:dancer:

File: src/components/containers/SubplotAccordion.js (hunk 1 above)
Comment (dmt0) [reply src/components/containers/SubplotAccordion.js]: that's what I was thinking when I was writing it lol

**Pull request merged**